pkg/kubelet/kubeletconfig/checkpoint/download.go

```diff
@@ -182,7 +182,7 @@ func (r *remoteConfigMap) Download(client clientset.Interface, store cache.Store
 		cm, err = getConfigMapFromStore(store, r.source.ConfigMap.Namespace, r.source.ConfigMap.Name)
 		if err != nil {
 			// just log the error, we'll attempt a direct download instead
-			klog.ErrorS(err, "Kubelet config controller failed to check in-memory store for remoteConfigMap", "apiPath", r.APIPath())
+			klog.ErrorS(err, "kubelet config controller failed to check in-memory store for remoteConfigMap", "apiPath", r.APIPath())
```

Comment:
all output with `klog.ErrorS(err,` starts with upper case. It is a log statement, not the returned error that can be wrapped